app/templates/core-data.hbs

```diff
@@ -1,3 +1 @@
-<WithSidebar>
-  {{outlet}}
-</WithSidebar>
+<SidebarContainer />
```

Comment:
The component should not include the outlet, the outlet should be passed down to the component, so we can include more things

Reply:
I restored the components and templates to their previous structure (how you implemented the sidebar). The only thing i changed now is a added some media queries and styling. Let me know if there is still room for improvement.